db/deploy-calibration.sh

```diff
@@ -6,7 +6,10 @@ cd "$(dirname "$(readlink -f "$0")")"
 # Add monorepo's node_modules to the PATH
 PATH=../node_modules/.bin:$PATH
 
-if ! wrangler d1 list | grep -q filcdn-db; then
-  wrangler d1 create filcdn-db --env calibration
+DB=filcdn-calibration-db
+ENV=calibration
```

Comment:
Should I rewrite this as follows?


```suggestion
ENV=calibration
DB=filcdn-$ENV-db
```

Reply:
Done in [60861d2](https://github.com/filcdn/worker/pull/264/commits/60861d2c6e92819c9cbe1b357f01eb1097fa57d3)

Reply:
I think it's fine to keep it this way unless we also generate the database names inside `wrangler.toml` using some script to keep them in sync.